### ISSUE TITLE
Feature request: UI enhancement for asset spamming (asset stacking)

### ISSUE THREAD

**Issue by apolon81** [mtgred/netrunner#1549]
Implementing "stacking" of identical naked servers containing exactly one rezzed card would do a lot here I think.

Example: one Pad Campaign is already rezzed in server 3, the corp rezzes the second one in server 6. The second one disappears but 2x watermark gets rendered on the original pad in server 3. A toastr (I think that's what is used for alerts?) warning might be in order here to help draw attention to the stacking.

The server selection in "run" button ideally would include only the original server (3). Also successful run could be tricked to actually work on one of the "stacked" servers (6-th in this example), not the original one. This way if the runner trashes the card, the stack stays in the same place. Only after all "alter-ego" cards are gone, the run on the server 3 is "the real run" on the server 3.

Thoughts?

**Comment by erbridge** [comment]
Duplicate of #1279.
